packages/react/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,38 @@
+import { CSSProperties, MouseEvent, ReactNode } from 'react';
+import { X } from "phosphor-react";
+import '@compassion-gds/css/src/components/Modal/modal.css';
+
+export interface ModalProps {
+  title?: string,
```

Comment:
Let's make this a `ReactNode` as well, so people can pass a `string` or JSX to be rendered as title